app/views/shared/_menu.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="container"></div>
+<div class="menu-container">
+  <span class="logout">
+    <% if user_signed_in? %>
+      <%= button_to 'Logout', destroy_user_session_path, method: :delete, class: "logout-btn" %>
+    <% end %>
+  </span>
+  <ul class="menu">
+    <li><%= link_to "Recipes", root_path, class: "menu-btn" %></li>
+    <li><%= link_to "Public Recipes", public_recipes_path, class: "menu-btn" %></li>
+    <li><%= link_to "Foods", user_foods_path(current_user), class: "menu-btn" %></li>
+    <li><a class="menu-btn" href="#">About</a></li>
+  </ul>
```

Comment:
- [optional] I think you can make this `About` menu item to be functional in order to make it to be interactive. 👍